README.md

```diff
@@ -80,6 +80,19 @@ steps:
     rules: resolutions
 ```
 
+Specify `ignore-packages` to skip resolution validation entirely for certain packages. Optionally provide a newline separated list of package names here.
```

Comment:
```suggestion
Specify `ignore-resolutions` to skip resolution validation entirely for certain packages. Optionally provide a newline separated list of package names here.
```

Reply:
Silly copy paste. Thanks for picking up 👍